### DISCUSSION TITLE
Unlit textures looking washed out

### DISCUSSION THREAD

**Discussion by jamsutton** [google/model-viewer#3147] [Q&A] [answered]
Hi all,
I'm having an issue with my unlit textures in model viewer. I bake lighting into my textures for added realism, but when I load my glb files into model-viewer the textures seem to loose contrast and become flatter? I've read through issues on here and think it could be a bug where tone-mapping is being applied to the unlit texture?

Here's a screenshot of the unlit texture and how it displays in model-viewer. You can see the loss of definition, especially in the highlights. 

<img src="https://www.jamsutton.com/modelviewer.jpg">

Could there be a work-around?
Thank you,
Jam

**Comment by samaneh-kazemi** [comment, marked as answer]
Hi @jamsutton, thanks for reaching out. This is a known issue. We are currently applying tone mapping to unlit models, which we shouldn't.

**Reply by jamsutton** [reply]
Hi @samaneh-kazemi - thanks so much for your reply. Loving model-viewer. Do you have any idea when this bug could potentially be fixed? Thank you

**Reply by elalish** [reply]
It hasn't been high priority since unlit models aren't physically-based. Still, you're the second user to notice, so I should probably deal with it. Might be a simple fix.

**Reply by jamsutton** [reply]
Thanks Emmett, that would be amazing!

**Reply by elalish** [reply]
Fixed in #3152. Once this is merged, please verify in our editor (since it uses the latest master).

**Reply by jamsutton** [reply]
Thanks so much for fixing this, amazing work you guys are doing! Some of my unlit baked textures use emissive textures (luminance with a black base colour) and not the KHR_materials_unlit extension - The tone mapping is being applied to emissive textures - is there an easy way to fix this? Thanks so much.

**Reply by elalish** [reply]
hmm, that's getting in pretty weird territory. What does emissive get you that unlit baseColor does not? We may need to add custom tone mapping support eventually...

**Reply by jamsutton** [reply]
Custom tone mapping option would be great! I have an AR app and use a few different ways of displaying AR content, they display an emissive texture and don’t seem to support the unlit extension. What is the best way of applying the KHR_materials_unlit extension to my model? I’ve tried to find a guide but haven’t had much luck…

**Reply by elalish** [reply]
It's a pretty simple piece of JSON in  the glTF; the unlit spec  shows what to do. I'd recommend the glTF VSCode extension for hand editing.
